src/main/app.js

```diff
@@ -46,8 +50,13 @@ require("electron").app.on("ready", gpii.app.electronAppListener);
 // Override default behaviour - don't exit process once all windows are closed
 require("electron").app.on("window-all-closed", fluid.identity);
 
-/*
- ** Component to manage the app.
+/**
+ * A component to manage the app. Each subcomponent of the app (except for the first
```

Comment:
This shouldn't be necessary, and depends on a framework facility that will be withdrawn shortly. If a component makes use of another component's facilities, (e.g. factsManager -> installID) this should be expressed either through an explicit dependency, or else mediated via an event or options distribution